qt/widgets/common/test/AlgorithmProgress/AlgorithmProgressDialogPresenterTest.h

```diff
@@ -37,74 +67,43 @@ class AlgorithmProgressDialogPresenterTest : public CxxTest::TestSuite {
     delete suite;
   }
 
-  void setUp() override {
-    mockDialogView.reset();
-    // The mock view also creates the presenter, because
-    // so that is passes the correct type into the constructor
-    mockDialogView = std::make_unique<NiceMock<MockAlgorithmProgressDialogWidget>>();
-  }
+  void test_true() { TS_ASSERT(true); }
```

Comment:
Is this also a relic of debugging?

Reply:
Yes I thought I'd removed it but a rebase brought it back during conflict resolution :man_facepalming: 
Thanks for noticing